src/plugins/intel_cpu/src/nodes/roi_align.cpp

```diff
@@ -668,6 +675,14 @@ ROIAlign::ROIAlign(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine
         } else if (m == ngPoolingMode::AVG) {
             algorithm = Algorithm::ROIAlignAvg;
         }
+        const ngAlignedMode mAligned = roiAlign->get_aligned_mode();
+        if (mAligned == ngAlignedMode::ASYMMETRIC) {
+            alignedMode = ROIAlignedMode::ra_asymmetric;
```

Comment:
I think we can remove prefix "ra" if it's ROIAlign's abbreviation.

Reply:
Hi, Chenhu: yes, prefix "ra" is ROIAlign's abbreviation here. but because all 'intel_cpu' nodes shared the same namespace(there are no sub-namespace for each node), so if we removed this prefix, it will be conflicted with other 'asymmetric' which is also defined in this namespace.